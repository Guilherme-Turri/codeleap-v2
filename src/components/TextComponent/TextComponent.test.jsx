import { screen } from '@testing-library/react'
import {TextComponent} from '.'
import { componentRender } from '../../render/componentRender'

describe('<TextCompoenent />', ()=>{
  it('should render on screen', ()=>{
    const {container } = componentRender(<TextComponent>test</TextComponent>)
    expect(screen.getByText('test')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  })
})
