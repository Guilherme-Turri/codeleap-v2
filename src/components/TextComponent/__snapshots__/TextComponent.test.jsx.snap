// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextCompoenent /> should render on screen 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div>
  <p
    class="c0"
  >
    test
  </p>
</div>
`;
